templates/plugin/ansible.cfg.erb

```diff
@@ -1,3 +1,4 @@
 [defaults]
 callback_whitelist = foreman
 local_tmp = <%= @working_dir %>
+host_key_checking = <%= @host_key_checking %>
```

Comment:
How about using booleans and just use ```@host_key_checking ? 'True' : 'False'``` to format it? Probably easier and I think kafo also has some issues with custom data types.